lib/cn/kvset.c

```diff
@@ -2539,15 +2534,12 @@ kvset_iter_enable_mblock_read(struct kvset_iterator *iter)
     struct kblk_reader *kr = &iter->kreader;
     struct vblk_reader *vr;
     void *              mem;
-    uint                vr_buf_sz;
-    uint                vb_max_sz;
+    uint64_t            vr_buf_sz;
     uint                i;
     merr_t              err;
-    int                 ra_size;
+    uint64_t            ra_size;
```

Comment:
read-ahead size should never be more than 1048576.  but i haven't looked at this code in a very long time...

Reply:
I am currently just basing it off of cn_compact_{k,v}blk_ra which is a uint64_t. Happy to reduce the type in kvs_rparams.c though.

Reply:
size_t is probably the right type, does config support size_t?  That said, size_t only for correctness in type, it was never expected be higher than 1048576 in practice.  Still looking at the code...  (int was probably always wrong!)

Reply:
Happy to add support for size_t as a part of this PR

Reply:
I'm all for being more standard on scalar types, but let's do that in PRs dedicated to that task.